spec/API_specification/creation_functions.md

```diff
@@ -76,7 +76,7 @@ Convert the input to an array.
 
 -   **copy**: _bool_
 
-    -   Whether or not to make a copy of the input. If `True`, always copies. If `False`, reuses existing memory buffer if possible. Default: `False`.
+    -   Whether or not to make a copy of the input. If `True`, always copies. If `False`, never copies. If `None`, reuses existing memory buffer if possible. Default: `None`.
```

Comment:
I think it is _probably_ a good idea to specify the specific exception (i.e., `ValueError`) that is raised when an otherwise valid input would require a copy.

Reply:
We've tried to avoid that in other places, but this particular may be so often needed when writing library code with `copy=False` that it makes sense to add.

Reply:
Added: "Must raise `ValueError` if copying of inputs which support DLPack or the buffer protocol is necessary and `copy=False`."

Data in sequence input is still copied of course.